packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/editingcontext/EditingContext.java

```diff
@@ -36,9 +40,12 @@ public class EditingContext implements IEMFEditingContext {
 
     private final List<View> views;
 
+    private final Map<String, List<Notification>> changes = Collections.synchronizedMap(new LinkedHashMap<>());
```

Comment:
EMF is not thread safe so this map shouldn't need to be thread safe just like the other attribute of this class.